js/src/lib/components/InferenceWidget/shared/WidgetWrapper/WidgetWrapper.svelte

```diff
@@ -67,8 +70,7 @@
 
 <div
 	class="flex w-full max-w-full flex-col
-	{isMaximized ? 'fixed inset-0 z-20 bg-white p-12' : ''}
-	{!modelLoadInfo ? 'hidden' : ''}"
+	{isMaximized ? 'fixed inset-0 z-20 bg-white p-12' : ''}"
```

Comment:
this undoes something recently added in #935

^^ cc @julien-c @gary149, I think there's been some discussion about the disappearance of the widget while its state is loading, this would bring it back - @mishig25 are there any side-effects/downsides in not hiding the widget if we don't have the state yet?

(IIUC, with the other changes in this PR, `WidgetInfo` would still show nothing if no state information)

Reply:
What's the logic behind hiding the widget while it's loading ? From what I saw if only create some nasty blinking effect when the widget suddenly appears.

Reply:
@mishig25